packages/shopify-codemod/transforms/convert-default-export-objects-to-named-exports.js

```diff
@@ -0,0 +1,21 @@
+export default function convertDefaultExportObjectsToNamedExports({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+
+  return j(source)
+    .find(j.ExportDefaultDeclaration)
+    .forEach((nodePath) => {
+      const exportIndex = nodePath.parentPath.node.body.findIndex((prop) => prop.type === 'ExportDefaultDeclaration');
```

Comment:
It's probably more clear to do `nodePath.parentPath.node.body.indexOf(nodePath.node)`.